src/test/scala/workflow/DelegatingTransformerSuite.scala

```diff
@@ -1,29 +0,0 @@
-package workflow
```

Comment:
Did you mean to delete this file? Is there not some equivalent test we would still want to run?